# Creates a reference count issue

To run, use an installation of Python 3.11.0 compiled with the `--with-pydebug` option, install using `pip`, and run with `python3 -m force_rc_error`. 